### PR TITLE
doc: Updated README.md

### PR DESCRIPTION
**Pull request by joscollin**
Updated README.md with:

* Setting up Git
* hub command

Signed-off-by: Jos Collin <jcollin@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,7 +108,15 @@ concept behind the implementation
 
 Once you've successfully made some changes, built, ran, tested and pushed to your GitHub remote - then you should submit a pull request for the changes to be applied on the upstream repo.
 
-Use [GitHub's interface](https://help.github.com/articles/about-pull-requests/) to create new pull requests. Once submitted, the code will go through some automated tests, integration and peer-reviews.
+To create new pull requests:
+
+* Use [GitHub's interface](https://help.github.com/articles/about-pull-requests/)
+* Use [hub](https://hub.github.com/) command.
+  ```sh
+  $ hub pull-request -b Tendrl/ui:master -h <Github-username>:<your-branch>
```
Comment (joscollin) [inline README.md]: I doubt if the base is `Tendrl/ui:master` or `Tendrl:master`. Please suggest.

Comment (sankarshanmukhopadhyay) [reply README.md]: should be the latter.

Comment (joscollin) [reply README.md]: @sankarshanmukhopadhyay Updated. Thanks.

**Comment by joscollin** [comment]
@sankarshanmukhopadhyay It looks like  `codacy/pr` never finishes. Is there a jenkins command to restart it ?

**Comment by sankarshanmukhopadhyay** [comment]
@joscollin might be worth pinging @r0h4n about this. Haven't seen it getting stuck previously.

**Comment by r0h4n** [comment]
The `codacy/pr` was a duplicate integration hook, I have removed it.

**Comment by joscollin** [comment]
Still doesn't finish.

**Comment by joscollin** [comment]
Closing this, because of no response.

**Pull request closed without merging**